frontend/src/components/data-outputs/data-output-technical-info/data-output-table-glue-columns.tsx

```diff
@@ -28,7 +28,7 @@ export const getGlueTechnicalInformationColumns = ({ t }: Props): TableColumnsTy
             render: (_, { data_output }) => {
                 const configuration: GlueDataOutputContract = data_output.configuration as GlueDataOutputContract;
                 // TODO figure out how to use product aligned databases here and get their ARN?
-                return configuration.database;
+                return configuration.database + '__' + configuration.database_suffix;
```

Comment:
Maybe it's better to use a [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) instead of summation here?

```suggestion
                return `${configuration.database}__${configuration.database_suffix}`;
```